src/app/context/SesionContext/useSession.js

```diff
@@ -0,0 +1,18 @@
+import React, { useState, useEffect } from 'react'
+import { auth } from 'app/services/Firebase/firebase'
+
+export const AuthContext = React.createContext()
+
+export const AuthProvider = ({ children }) => {
+  const [currentUser, setCurrentUser] = useState(null)
+  useEffect(() => console.log(currentUser), [currentUser])
```

Comment:
I think it will be better to rename file as authContext and remove this useEffect with console.log